crates/bitwarden-crypto/src/enc_string/symmetric.rs

```diff
@@ -237,6 +237,10 @@ impl KeyDecryptable<SymmetricCryptoKey, Vec<u8>> for EncString {
     fn decrypt_with_key(&self, key: &SymmetricCryptoKey) -> Result<Vec<u8>> {
         match self {
             EncString::AesCbc256_B64 { iv, data } => {
+                if key.mac_key.is_some() {
+                    return Err(CryptoError::InvalidKey);
```

Comment:
I think we should make an explicit error about this to provide more appropriate error messages.

Maybe `KeyMissingMac`?